ambari-server/src/main/java/org/apache/ambari/server/stack/StackModule.java

```diff
@@ -1156,7 +1156,7 @@ private UpgradePack parseServiceUpgradePack(UpgradePack parent, File serviceFile
    * @return the child upgrade pack
    */
   private UpgradePack parseServiceUpgradePack(UpgradePack parent, UpgradePack child) {
-    parent.mergePrerequisiteChecks(child);
+    // parent.mergePrerequisiteChecks(child);
```

Comment:
Any reason not to just get rid of it?

Reply:
Because I forgot about it about 2 seconds after I was off the file while I was looking for my Guice issue.  Will fix :)